.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Release
+
+on:
+  release:
+    types: published
+
+jobs:
+  package_release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.x
+
+    - name: Build package
+      run: |
+        pip install wheel
+        python setup.py sdist bdist_wheel
+
+    - name: Publish package
+      uses: pypa/gh-action-pypi-publish@master
+      with:
+        user: __token__
+        password: ${{ secrets.pypi_token }}
```

Comment:
That'd be Thermondo token, right? We were still using Johannes's credentials here...

Reply:
@amureki yes, that is a generated token out of our thermondo pypi account, with access only to this repo

Reply:
@codingjoe fyi :) when this is merged we won't use your pypi account any more :)

Reply:
FYI, I'd recommend using twine directly, not the docker image from PyPa. They also just use twine, but it's obviously a lot slower to start yet another container. I would also recommend updating setuptools, since it's not always the latest version. I usually also add a little readme-render check, something like:
```
python3 -m pip install --upgrade pip setuptools wheel twine readme-renderer
python3 setup.py sdist bdist_whell
python -m twine check dist/*
python -m twine upload dist/*
```

Reply:
@codingjoe good points, I assumed the pypa action would be a simpler JS one and not using a docker image. 

I'll check it out